src/vm/arm64/asmconstants.h

```diff
@@ -89,7 +89,10 @@ ASMCONSTANTS_C_ASSERT(MachState__isValid == offsetof(MachState, _isValid))
 #define LazyMachState_captureX19_X29 MachState__captureX19_X29
 ASMCONSTANTS_C_ASSERT(LazyMachState_captureX19_X29 == offsetof(LazyMachState, captureX19_X29))
 
-#define LazyMachState_captureSp     (MachState__isValid+8) // padding for alignment
+#define LazyMachState_captureFp     (MachState__isValid+8) // padding for alignment
+ASMCONSTANTS_C_ASSERT(LazyMachState_captureFp == offsetof(LazyMachState, captureFp))
+
+#define LazyMachState_captureSp     (LazyMachState_captureFp+8)
```

Comment:
We don't need CaptureFP -- the capture helper doesn't modify the frame pointer.
So, the FP value stored in capturex19_x29[10] is current.